src/components/Header/mobile-menu/index.tsx

```diff
@@ -10,11 +11,27 @@ type Props = {
   whiteBg: boolean;
 };
 
-export const MobileMenu: React.FC<Props> = ({ toggleMenu, isMenuOpened, menuItems, whiteBg }) => {
+export const MobileMenu: React.FC<Props> = ({
+  toggleMenu,
+  isMenuOpened,
+  menuItems,
+  whiteBg,
+}) => {
   return (
     <>
       <MenuTrigger toggleMenu={toggleMenu} isMenuOpened={isMenuOpened} />
-      {isMenuOpened && <MenuList menuItems={menuItems} whiteBg={whiteBg} />}
+      {isMenuOpened && (
+        <>
+          <Helmet>
+            <body className="overlay-locked" />
+          </Helmet>
```

Comment:
I suggest replacing it with "react-remove-scroll-bar".
It is pretty simple to use, look for example [here](https://github.com/JetBrains/kotlin-web-site-ui/pull/25/files#diff-8197f2959ca8cf119392eafe2e26e1a84a2db43c4c9846ea73e8c0c4ed3fcbd5R5).
There are two reasons to do this:
- `overflow: hidden` will scroll page to the top and I guess that on iOS it still scrollable.
- I going to use Helmet on the Layout component, this may cause a conflict.

Reply:
Ok, will fix